internal/kubernetes/operator/features/crds.go

```diff
@@ -71,9 +71,18 @@ var (
 	}
 )
 
+func GetResourcesForVersion(version string) ([]string, bool) {
+	semVer := strings.Split(version, ".")
+	semVer[2] = "0"
+	majorVersion := strings.Join(semVer, ".")
+
+	resources, ok := versionsToResourcesMap[majorVersion]
+	return resources, ok
+}
```

Comment:
we have https://github.com/Masterminds/semver, would it help here?

Reply:
yes, we can use it. I made the changes